packages/ember-data/lib/adapters/build_url_mixin.js

```diff
@@ -0,0 +1,111 @@
+var get = Ember.get;
+
+export default Ember.Mixin.create({
```

Comment:
@bmac ok, I've added this, and some documentation for how to use `BuildURLMixin`. It is not very thorough, but my goal is to completely change the way url building is done (with url templates), and I don't want to spend too much time improving documentation for something that will soon be obsolete.